english/dream/dream.md

```diff
@@ -5,3 +5,5 @@ You feel that there will be no consequence in your dream and you attempt to mani
 Afterwards, you decide to:
 
 [scream](../scream/scream.md)
+
+[lucid dream](../lucid/lucid.md)
```

Comment:
@SamuraiRanderson, it looks like you may have forgotten to commit the `lucid.md` file in this pull request. Also, if you would like this pull request merged instead of your previous one, could you close the other one?